### DISCUSSION TITLE
[建议] 增加存过/函数创建

### DISCUSSION THREAD

**Discussion by peng19832** [hhyo/Archery#1630] [Ideas]
书接上回 #1626 
goinception不支持。
可不可以，不通过goinception创建存过/函数呢？
比如：检测语句时，如果检测到create procedure/create function这类关键词，简单判断下语法格式正确后，原文交给后端实例执行，
然后抓取反馈报错信息，返回页面。

**Comment by LeoQuote** [comment]
可以开手工确认, 走线下执行: https://github.com/hhyo/archery/wiki/configuration#mannual
